test/e2e/tests/tokens/custom-token-send-transfer.spec.js

```diff
@@ -118,7 +117,7 @@ describe('Transfer custom tokens @no-mmi', function () {
         // transfer token from dapp
         await openDapp(driver, contractAddress);
         await driver.clickElement({ text: 'Transfer Tokens', tag: 'button' });
-        await switchToNotificationWindow(driver);
+        await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
```

Comment:
Why does this improve flakiness? What is the cause of the issue being solved?

Reply:
This was just refactoring the deprecated method, mainly there is no real flakiness on this test. Multiple re runs of it don't show any.